Tests/Controllers/CompaniesController.cs

```diff
@@ -13,9 +10,27 @@ namespace Tests.Controllers
     public class CompaniesController : ApiController
     {
         [HttpGet]
+        [Route("companies/{id}")]
         public Company GetCompany(string id)
         {
             return new Company(prefill: true);
         }
+
+        [HttpGet]
+        [Paginated(PerPage = 12)]
+        [Route("companies")]
+        public IEnumerable<Company> GetCompanies()
+        {
+            return GetEnumerable().Take(100);
+        }
+
+        private static IEnumerable<Company> GetEnumerable()
```

Comment:
This should probably be a method on a static helper class, since now it's duplicated a lot.